roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+  Interface to OpenShift oc command
+"""
+#
+#   Copyright 2015 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import os
+import shlex
+import atexit
+import shutil
+import string
+import random
+import subprocess
+import yaml
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+# pylint: disable=bare-except
+def cleanup_file(inc_file):
+    """ clean up """
+    try:
+        os.unlink(inc_file)
+    except:
+        pass
+
+
+class OCUtil(object):
```

Comment:
If we are going to move this module from [`openshift-tools`](https://github.com/openshift/openshift-tools/blob/630252b138a658bec2ed9305e86edced1688a956/openshift_tools/monitoring/ocutil.py), perhaps we should consider updating the license date, and proposing the monitoring code use it from here?

We don't want two copies, but we need to settle the location where it should live.